logstash-core-event-java/src/main/java/com/logstash/ext/JrubyEventExtLibrary.java

```diff
@@ -274,5 +286,12 @@ public IRubyObject ruby_tag(ThreadContext context, RubyString value)
         public IRubyObject ruby_timestamp(ThreadContext context) throws IOException {
             return new JrubyTimestampExtLibrary.RubyTimestamp(context.getRuntime(), this.event.getTimestamp());
         }
+
+        @JRubyMethod(name = "logger=", required = 1, meta = true)
+        public static IRubyObject ruby_set_logger(ThreadContext context, IRubyObject recv, IRubyObject value)
+        {
+            Event.setLogger(value.isNil() ? null : new ProxyLogger((RubyObject)value));
```

Comment:
should we fail this if it is `null`? I understand that the current behavior internally is to default to the default logger (stdio), but technically this operation failed since it could not set a new logger.